### PR TITLE
(feat) : KHP3-6307 Add patient-chart in-patient view

### PR DESCRIPTION
**Pull request by donaldkibet**
## Description

see https://thepalladiumgroup.atlassian.net/browse/KHP3-6307

## Screenshot
https://github.com/user-attachments/assets/d41e6173-6457-4d1f-9141-b4b227aa9b02

### CODE REVIEW

File: src/config-schema.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const configSchema = {
     _type: Type.UUID,
     _description:
       "UUID for the location tag of the `Admission Location`. Patients may only be admitted to inpatient care in a location with this tag",
-    _default: "839c65c7-9998-4b90-b80b-39727dfe9fa2",
+    _default: "233de33e-2778-4f9a-a398-fa09da9daa14",
```
Comment (ojwanganto) [inline src/config-schema.ts:9]: We'll later on transition to using tag name in place of UUID. Aligning to UUIDs can sometimes be very hectic

File: src/config-schema.ts [hunk 2]
```diff
@@ -24,4 +24,34 @@ export const configSchema = {
       "Endpoint for fetching list of patients eligible for ward admission",
     _default: "",
   },
+  inPatientForms: {
+    _type: Type.Array,
+    _description: "List of forms that can be filled out for in-patients",
+    _default: [
```
Comment (ojwanganto) [inline src/config-schema.ts:30]: We can couple this with additional logic to control which patient can be filled which form. May be an endpoint which then evaluates the forms and returns only the appropriate ones will help. For now we'll go with this as is

**Comment by ojwanganto** [comment]
Thanks, @donaldkibet. Can we move this feature to the kenyaemr-3.x? We may still end up keeping this alongside what the community has for a while.

**Comment by donaldkibet** [comment]
closing this to migrate this work to kenyaemr-esm-3.x repo

**Pull request closed without merging**